crates/aptos/src/update/update_helper.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    cli_build_information,
+    update::{get_additional_binaries_dir, UpdateRequiredInfo},
+};
+use anyhow::{anyhow, bail, Context, Result};
+use aptos_build_info::BUILD_OS;
+use self_update::{backends::github::Update, update::ReleaseUpdate};
+use std::path::PathBuf;
+
+#[cfg(target_arch = "x86_64")]
+pub fn get_arch() -> &'static str {
+    "x86_64"
+}
+
+#[cfg(target_arch = "aarch64")]
+pub fn get_arch() -> &'static str {
+    "aarch64"
+}
+
+#[cfg(target_arch = "arm")]
+pub fn get_arch() -> &'static str {
+    "arm"
+}
+
+#[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64")))]
```

Comment:
```suggestion
#[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64", target_arch = "arm")))]
```
Shouldn't it be this given there is a `get_arch` implementation for `arm`.